src/main/java/com/linkedin/replica/serachEngine/controller/Server.java

```diff
@@ -83,6 +85,13 @@ public void operationComplete(ChannelFuture ch) throws Exception {
 			workerGroup.shutdownGracefully();
 			bossGroup.shutdownGracefully();
 		}
-        
+    
+	}
+	
+	public static void main(String[] args) throws InterruptedException, IOException {
```

Comment:
Do we need this main method? we should start the controller from the project main right? Also, the host and port should be read from the config file.

Reply:
I was testing the controller netty server only. i will remove it